docs/resources/project.md

```diff
@@ -1,45 +1,97 @@
-# Artifactory Project Resource
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "project Resource - terraform-provider-projects"
+subcategory: ""
+description: |-
+  Provides an Artifactory project resource. This can be used to create and manage Artifactory project, maintain users/groups/roles/repos.
+---
+
+# project (Resource)
 
 Provides an Artifactory project resource. This can be used to create and manage Artifactory project, maintain users/groups/roles/repos.
 
 ## Example Usage
 
-```hcl
-# Create a new Artifactory project called my-project
-resource "project" "my-project" {
-  key                      = "myprj"
-  display_name             = "My Project"
-  max_storage_in_gibabytes = 10
-
+```terraform
+resource "project" "myproject" {
+  key = "myproj"
+  display_name = "My Project"
+  description  = "My Project"
   admin_privileges {
     manage_members   = true
     manage_resources = true
     index_resources  = true
   }
+  max_storage_in_gibabytes   = 10
+  block_deployments_on_limit = false
+  email_notification         = true
 
   member {
     name  = "user1"
     roles = ["developer","project admin"]
   }
+
+  member {
+    name  = "user2"
+    roles = ["developer"]
+  }
+
+  group {
+    name = "dev-group"
+    roles = ["developer"]
+  }
+
+  group {
+    name = "release-group"
+    roles = ["release manager"]
+  }
 }
 ```
 
-## Argument Reference
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- **admin_privileges** (Block Set, Min: 1) (see [below for nested schema](#nestedblock--admin_privileges))
+- **display_name** (String) Also known as project name on the UI
+- **key** (String) The Project Key is added as a prefix to resources created within a Project. This field is mandatory and supports only 3 - 6 lowercase alphanumeric characters. Must begin with a letter. For example: us1a.
+
+### Optional
+
+- **block_deployments_on_limit** (Boolean)
```

Comment:
description here

Reply:
I will add to the schema. Nothing from UI so I'll have to be creative 😄